engine/admin/logging/awslogs.md

```diff
@@ -77,6 +91,38 @@ $ docker run --log-driver=awslogs \
 > **Note**:
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.
 
+### aws-datetime-format
+
+The `aws-datetime-format` option defines a multiline start pattern in [Python
+`strftime` format](http://strftime.org). This option always takes precedence if
+both `awslogs-datetime-format` and `awslogs-multiline-pattern` are configured.
+
+For example:
+
+```bash
+$ docker run -it --rm \
+  --log-driver=awslogs \
+  --log-opt awslogs-group=test \
+  --log-opt awslogs-datetime-format='%Y-%m-%d' \
+  awslogtest:latest /test2.sh
+```
+
+### aws-multiline-pattern
+
+The `aws-multiline-pattern` option defines a multiline start pattern using a
+regular expression. This option is ignored if `awslogs-datetime-format` is also
+configured.
```

Comment:
Maybe we should mention what this is useful for, or do you think it's clear from this? (usage can be an application that outputs, e.g. a stack-dump, which would otherwise be logged in multiple entries, and now allows you to capture it in a single log-entry. (could be better examples though)